src/app/public/modules/vertical-tabset/vertical-tab.component.ts

```diff
@@ -40,6 +40,7 @@ import {
   styleUrls: ['./vertical-tab.component.scss'],
   providers: [
     SkyVerticalTabMediaQueryService,
+    SkyVerticalTabsetAdapterService,
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```